components/x-podcast-launchers/src/PodcastLaunchers.jsx

```diff
@@ -53,30 +64,36 @@ class PodcastLaunchers extends Component {
 
 	render() {
 		const { rssUrl } = this.state;
-		return (
+		const { conceptId, conceptName, csrfToken, isFollowed} = this.props
+		const renderFollowButton = typeof renderFollowButton === 'function' ? renderFollowButton : defaultFollowButtonRender;
+
+		return rssUrl && (
 			<div className={styles['container']}>
-				{rssUrl && (
-					<div>
-						<ul className={styles['podcast-app-links__wrapper']}>
-						{generateAppLinks(rssUrl).map(({ name, url }) => (
-							<li key={name}>
-								<a href={url} className={podcastAppLinkStyles}>{name}</a>
-							</li>
-						))}
-						</ul>
-						<div className={rssUrlWrapperStyles}>
-							<input className={rssUrlInputStyles} value={rssUrl} type='text' readOnly/>
-							<div className={rssUrlCopyButtonWrapperStyles}>
-								<button
-									className={basicButtonStyles}
-									onClick={copyToClipboard}
-									data-url={rssUrl}>
-									Copy RSS
-								</button>
-							</div>
+				<h2 className={styles['heading']}>Subscribe on a podcast app</h2>
+				<div>
```

Comment:
I think given we have these headings now, we might not need this extra wrapping `div`?

Reply:
👍